_docs/development/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Overview
+order: 001
```

Comment:
Within a category, the `order` controls where the page is displayed on the `#left-sidebar` navigation menu. Furthermore, within a category, there can be subcategories, e.g., "Install Locally" and "Install on GCP" exist under a general "Install" sub-category which exists under "How To" category so... "How To" > "Install" > "Install Locally"... each having a respective page.

With that said, `order` is used to control sorting within a sub-category as well. But the mapping is flat and can't be constructed hierarchically without a plugin (which GitHub Pages does not support), so you have to flatten the two orders into one integer. Thus you end up with say:
```
"Overview" order: 101
"Install" order: 201
    "Install Locally" order: 202
    "Install GCP" order: 203
"Another Article" order: 301
```
The first N digits (N=1 in this case) representing the order in the first-level and the last M digits (M=2) representing the order in the second-level of the hierarchy.

Another solution would be to break these out into two separate integers: `category_order` and `subcategory_order` say, and then flatten them in the template by sorting with something like `1000 * category_order + subcategory_order`. That's also an option.

What're your thoughts?